server/src/resolvers/getVariantsResolver/utils/liftOver.ts

```diff
@@ -0,0 +1,93 @@
+import { AssemblyId, VariantQueryDataResult } from '../../../types';
+import resolveAssembly from '../utils/resolveAssembly';
+const fs = require('fs/promises');
+const tmpdir = require('os').tmpdir;
+const path = require('path');
+const promiseExec = require('util').promisify(require('child_process').exec);
```

Comment:
I'm pretty sure because you're using TypeScript, these should be `import` statements instead, and let the compiler convert these to CJS `require`

Reply:
Agreed! You can do 

`import * as fs from 'fs'` for example